Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/ColorTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
+{
+    public class ColorTests : TypeTester<Color>
+    {
+        public static (Color deserialized, string serialized)[] representations = new[] {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'representations' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4930242)